contracts/StrategyCurveTwoPool.sol

```diff
@@ -0,0 +1,383 @@
+        gaugeFactory.mint(address(gauge));
+    }
+
+    function claimRewards() external onlyVaultManagers {
+        // Claims any pending CRV
+        //
+        // Mints claimable CRV from the factory gauge. Reward tokens are sent to `msg.sender`
+        // The method claim_rewards() from the old gauge now only applies to third-party tokens.
+        // There are no third-party tokens in this strategy.
+        _claimRewards();
+    }
+
+  function prepareMigration(address _newStrategy) internal override {
+        // Withdraw LP tokens from the gauge. The transfer to the new strategy is done
+        // by migrate() in BaseStrategy.sol. Note that this function does not claim
+        // any pending rewards. To claim and transfer pending rewards to a new
+        // strategy, please use claimAndTransferRewards()
+    uint256 _stakedBal = stakedBalance();
+    if (_stakedBal > 0) {
+      gauge.withdraw(_stakedBal);
+    }
+  }
+
+  function protectedTokens()
+    internal
+    view
+    override
+    returns (address[] memory)
+  {}
+
+  /* ========== SETTERS ========== */
+
+  // These functions are useful for setting parameters of the strategy that may need to be adjusted.
+
+  // Credit threshold is in want token, and will trigger a harvest if credit is large enough.
+  function setCreditThreshold(uint256 _creditThreshold)
+    external
+    onlyEmergencyAuthorized
+  {
+    creditThreshold = _creditThreshold;
+  }
+
+  // This allows us to manually harvest with our keeper as needed
+  function setForceHarvestTriggerOnce(bool _forceHarvestTriggerOnce)
+    external
+    onlyEmergencyAuthorized
+  {
+    forceHarvestTriggerOnce = _forceHarvestTriggerOnce;
+  }
+
+  /* ========== KEEP3RS ========== */
+
+  /// @param callCostInWei Cost of the contract call to harvest(). Used to determine if the profit (if any) is
+  ///   sufficiently large to justify the call to harvest()
+  function harvestTrigger(uint256 callCostInWei)
+    public
+    view
+    override
+    returns (bool)
+  {
+    // Should not trigger if strategy is not active (no assets and no debtRatio). This means we don't need to adjust keeper job.
+    if (!isActive()) {
+      return false;
+    }
+
+    StrategyParams memory params = vault.strategies(address(this));
+
+    // harvest no matter what once we reach our maxDelay
+    if (block.timestamp.sub(params.lastReport) > maxReportDelay) {
+      return true;
+    }
+
+    // harvest our credit if it's above our threshold
+    if (vault.creditAvailable() > creditThreshold) {
+      return true;
+    }
+
+    // trigger if we want to manually harvest
+    if (forceHarvestTriggerOnce) {
+      return true;
+    }
+
+    // otherwise, we don't harvest
+    return false;
+  }
+
+  // convert our keeper's eth cost into want, we don't need this anymore since we don't use baseStrategy harvestTrigger
+  function ethToWant(uint256 _ethAmount)
+    public
+    view
+    override
+    returns (uint256)
+  {
+    return _ethAmount;
```

Comment:
Can delete this

Reply:
will do

Reply:
`return _ethAmount;` can be deleted since you don't use it anywhere